recipes/or-tools/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -BUILD_DEPS=OFF \
+      -DUSE_SCIP=OFF \
+      -USE_SCIP=OFF \
```

Comment:
why this one ? `-DUSE_SCIP=OFF` should be enough

Reply:
I had tried only `-USE_SCIP=OFF` which didn't work but let's see if just `-DUSE_SCIP=OFF` works.